utils/json/src/address.rs

```diff
@@ -124,29 +124,11 @@ mod tests {
 
     use super::*;
 
-    #[cfg(test)]
-    #[derive(Clone, Debug, PartialEq)]
-    struct AddressWrapper {
-        address: Address,
-    }
-
-    #[cfg(test)]
-    impl quickcheck::Arbitrary for AddressWrapper {
-        fn arbitrary(g: &mut quickcheck::Gen) -> Self {
-            let address: fvm_shared::address::Address = arbitrary::Arbitrary::arbitrary(
-                &mut arbitrary::Unstructured::new(&Vec::arbitrary(g)),
-            )
-            .unwrap();
-            AddressWrapper {
-                address: Address::from(address),
-            }
-        }
-    }
-
     #[quickcheck]
-    fn address_roundtrip(address: AddressWrapper) {
-        let serialized = serde_json::to_string(&json::AddressJsonRef(&address.address)).unwrap();
-        let parsed: json::AddressJson = serde_json::from_str(&serialized).unwrap();
-        assert_eq!(address.address, parsed.0);
+    fn address_roundtrip(id: u64) {
+        let address = Address::new_id(id);
```

Comment:
`Address_v3` _does_ implement `Arbitrary` so you can use it directly:

```suggestion
    fn address_roundtrip(address: Address_v3) {
        let address: Address = address.into();
```

Now, you shouldn't really do this. It would be nicer if you implement `Arbitrary` for `Address` instead.